src/common/reducers/auth-error.js

```diff
@@ -1,9 +1,12 @@
 import { AUTH_ERROR } from '../actions/auth-error';
 
-export function authError(state = [], action) {
+export function authError(state = {}, action) {
```

Comment:
I guess the array initially came from MU (https://github.com/canonical-ols/my.ubuntu.com/blob/master/src/common/reducers/oyez.js#L3) - where we possibly show multiple status messages rendered by server. So the idea here was to support showing multiple error messages.

But as long as it fixed the issue we have now I guess it's good to consistently treat `authError` as an object.